kishonStClair.js

```diff
@@ -0,0 +1,28 @@
+const menu = {
+    pepperoni: {
+      toppings: ["garlic", "tomatos", "pepperoni"],
+      price: "$5"
+    },
+    cheese: {
+      toppings: ["garlic", "tomatos"],
+      price: "$5"
+    },
+    Vanilla: {
+      toppings: ["garlic", "oregano", "pepperoni"],
+      price: "$8"
+    },
+    mediterrean: {
+      toppings: ["garlic", "Olives", "pineapple"],
```

Comment:
tiny detail, honestly doesn't matter too much, but be consistent with your capitalization! these small details don't matter too much right now, but when you're working with bigger code bases, it can really mess up how you keep your variables straight